fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -194,6 +194,15 @@ class TabsTrayFragment : AppCompatDialogFragment() {
         return tabsTrayDialog
     }
 
+    override fun onPause() {
+        super.onPause()
+
+        // On some android versions (known about OneUI 2.5) entering animation is played every time
+        // DialogFragment goes though an onStart() lifecycle call. We want to prevent animation from
+        // being repeated after app returns from background.
```

Comment:
nit: 
```suggestion
        // On some android versions (known about OneUI 2.5) entering animation is played every time
        // DialogFragment goes though an onStart() lifecycle call. We want to prevent animations from
        // being repeated after app returns from background.
```

I also think that this comment could live next to the style you've defined instead. That way, if we end up re-using it in other dialogs, the comment will be in the common area.